doc/supportedPlatformVerions.md

```diff
@@ -1,67 +1,22 @@
 # Supported platforms and versions
 
-## Supported stretch versions
+## dotnet
 
-### dotnet
```

Comment:
we still want to keep these versions for documentation, could you revert the changes in this file

Reply:
yeah thanks .. i will undo this change